### PR TITLE
virsh_domtime: Add cases for 'readonly' test

### PR DESCRIPTION
**Pull request by hahakiki2010**
Signed-off-by: Yan Li <yannli@redhat.com>

### CODE REVIEW

**Comment by hahakiki2010** [comment]
Test passed
(virtenv_py3) [root@localhost domain]# avocado run --vt-type libvirt virsh.domtime.negative.readonly_test.get_time_readonly
JOB ID     : 005185e460ac8107ad20ff8b00fa9e173f777909
JOB LOG    : /root/avocado/job-results/job-2018-07-19T15.32-005185e/job.log
 (1/1) type_specific.io-github-autotest-libvirt.virsh.domtime.negative.readonly_test.get_time_readonly: PASS (27.38 s)
RESULTS    : PASS 1 | ERROR 0 | FAIL 0 | SKIP 0 | WARN 0 | INTERRUPT 0 | CANCEL 0
JOB TIME   : 37.66 s
(virtenv_py3) [root@localhost domain]# avocado run --vt-type libvirt virsh.domtime.negative.readonly_test.set_time_readonly
Failed to load plugin from module "avocado.plugins.journal": ModuleNotFoundError("No module named '_sqlite3'",)
Failed to load plugin from module "avocado.plugins.distro": ModuleNotFoundError("No module named '_bz2'",)
Failed to load plugin from module "avocado.plugins.journal": ModuleNotFoundError("No module named '_sqlite3'",)
JOB ID     : 176b334cd0bc2fb3c16ee123b05b2092d1583a99
JOB LOG    : /root/avocado/job-results/job-2018-07-19T15.33-176b334/job.log
 (1/1) type_specific.io-github-autotest-libvirt.virsh.domtime.negative.readonly_test.set_time_readonly: PASS (29.06 s)
RESULTS    : PASS 1 | ERROR 0 | FAIL 0 | SKIP 0 | WARN 0 | INTERRUPT 0 | CANCEL 0
JOB TIME   : 39.24 s

File: libvirt/tests/src/virsh_cmd/domain/virsh_domtime.py [hunk 1]
```diff
@@ -357,7 +361,7 @@ def stop_vm():
 
             # Run command with specified options.
             res = virsh.domtime(vm_name, now=now, pretty=pretty, sync=sync,
-                                time=set_time)
+                                time=set_time, readonly=readonly, debug=True)
```
Comment (chunfuwen) [inline libvirt/tests/src/virsh_cmd/domain/virsh_domtime.py]: do we really need debug= True?

**Comment by iccaszhulili** [comment]
rh-caselink retest please

File: libvirt/tests/cfg/virsh_cmd/domain/virsh_domtime.cfg [hunk 2]
```diff
@@ -49,3 +49,9 @@
                     domtime_time = -1
                 - set_time_too_large:
                     domtime_time = 9223372037
+                - readonly_test:
+                    readonly_test = yes
+                    variants:
+                        - get_time_readonly:
```
Comment (balamuruhans) [inline libvirt/tests/cfg/virsh_cmd/domain/virsh_domtime.cfg]: `get_time_readonly` should work with readonly and it should be positive test case right?

File: libvirt/tests/cfg/virsh_cmd/domain/virsh_domtime.cfg [hunk 3]
```diff
@@ -49,3 +49,9 @@
...
+                        - set_time_readonly:
+                            domtime_pretty = yes
```
Comment (balamuruhans) [inline libvirt/tests/cfg/virsh_cmd/domain/virsh_domtime.cfg]: `domtime_time = "1334567890"` should be given for `set_time_readonly`

**Pull request merged**